python/requirements.txt

```diff
@@ -18,3 +18,4 @@ statsd==3.2.2
 progressbar2==3.35.2
 ijson==2.3
 gitpython==2.1.8
+freezegun==0.3.10
```

Comment:
I think freezegun is being used in the previous commit, and it would be good to add the requirement there also.